### DISCUSSION TITLE
Github awful way of sending academic proof

### DISCUSSION THREAD

**Discussion by juanaugusto0** [community/community#143409] [GitHub Education]
### Hi everyone,

Hi everyone, my name is Juan and im trying really hard to apply for Github Education to get the copilot, but the only way to send the image of my academic proof is A: Taking a photo on my notebook's webcam (the document is online so it's totally impossible) or B: Taking the photo with ONLY the front facing camera of my cellphone (obviously also impossible since i need to take the photo of my notebook screen and its already hard enough with the back facing camera), so i really need help from the support to have another way to send the proof

**Comment by davevad93** [comment]
Hi @juanaugusto0 ,

There should be an `Upload` button that allows to submit a photo proof or screenshot of your academic documentation, the thing is it doesn't always appear, there are many users who haven't been able to upload anything and I don't know why. My only advice is to try entering the [application page](https://education.github.com/discount_requests/application) with different browsers or devices and check if the upload button finally appears.
